services/wallet/transfer/transaction_manager_route.go

```diff
@@ -164,7 +164,8 @@ func (tm *TransactionManager) buildTxForPath(path *routes.Path, pathProcessors m
 
 		// special handling for transfer tx if selected token is not ETH
 		// TODO: we should fix that in the trasactor, but till then, the best place to handle it is here
-		if !path.FromToken.IsNative() {
+		// Paraswap needs the value to form the path key
+		if path.ProcessorName != pathprocessor.ProcessorSwapParaswapName && !path.FromToken.IsNative() {
```

Comment:
@dlipicar based on what I see here we have the same issue for bridging other than ETH tokens.

Reply:
This issue started after we've added `Amount` to the key (under which we're storing some data for processors) to increase the uniqueness and fix the bridging issue. 

Will try to think now about how to fix it globally.